Resources/Textures/Structures/Wallmounts/posters.rsi/meta.json

```diff
@@ -1 +1,149 @@
+{
+  "version": 1,
+  "license": "CC-BY-3.0",
+  "copyright": "https://github.com/cmss13-devs/cmss13/blob/HEAD/icons/obj/structures/props/posters.dmi",
```

Comment:
Incorrect copyright source, rsi edit on the latest version should do it automatically.

It would return `Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/1ea078e04c49e1004b0bb715f252260cdf39b602/icons/obj/structures/props/posters.dmi` in this case. I also forgot to mention you don't even need to download the .dmi, the rsi editor does the entire thing apparently. Thanks smugleaf!